devel/corepack/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           npm 1.0
+
+name                corepack
+version             0.15.0
+revision            0
+
+categories          devel
+license             MIT
+maintainers         @Robotex openmaintainer
+
+description         Zero-runtime-dependency package acting as bridge between Node projects and their package managers
+long_description    Corepack is a zero-runtime-dependency Node.js script that acts as a bridge between Node.js projects and the package managers they are intended to be used with during development
```

Comment:
maybe truncate line to make it easy to read?

Reply:
I haven't seen anywhere in the documentation what's the convention for Portfile line lengths, so I just truncated it to keep it below 120 characters

Reply:
Thanks! It seems amazing!